packages/web-app-files/src/App.vue

```diff
@@ -104,9 +104,8 @@ export default {
         revert: event === 'beforeDestroy'
       })
     },
-
-    $_ocApp_dragOver() {
-      this.dragOver(true)
+    $_ocApp_dragOver(event) {
+      this.dragOver((event.dataTransfer.types || []).some(e => e === 'Files'))
```

Comment:
This is super hard to read. The way it was before gave context that the `dragOver` is about uploading something. Ok for me to inline it, but please assign it to a constant with a reasonable name and then pass the constant into `dragOver`. Only for developer experience (DX). ;-)

Reply:
that was my fault, you're right more context is better 👍